src/langapi/language_util.cpp

```diff
@@ -8,15 +8,33 @@ Author: Daniel Kroening, kroening@cs.cmu.edu
 
 #include "language_util.h"
 
-#include <memory>
-
-#include <util/symbol_table.h>
+#include <util/invariant.h>
 #include <util/namespace.h>
 #include <util/std_expr.h>
+#include <util/symbol_table.h>
 
 #include "language.h"
 #include "mode.h"
 
+#include <memory>
```

Comment:
I had to reorder the includes section to match the current clang-format rules :

```
IncludeBlocks: Regroup
IncludeCategories:
  - Regex: '_class.h(>|")$'
    Priority: 0
  - Regex: '<util/'
    Priority: 1
  - Regex: '<goto-programs/'
    Priority: 2
  - Regex: '<.+/.+>'
    Priority: 3
  - Regex: '^"[^/]+"$'
    Priority: 4
  - Regex: '<[^/]+>'
    Priority: 5
```
As you see `#include <memory>` has priority 5 according to these rules.

Reply:
It's in that order on purpose (I can say so as this piece of code is my fault). My rationale for placing STL headers last is that it will help surface missing includes that would suddenly bite when a header file is newly used in another place. But maybe that's a bad idea?

Reply:
I have no objections to this new ordering and it's actually good to know if any other include file is inadvertently missing an stl include.

I was just surprised by global reordering effect of adding a single new line in the existing block of `#include <utils/*.h>`. Let's keep using the new ordering policy.

Reply:
Yes, it does result in some short-term churn, but should lead to eventual consistency. Previously, there were unwritten rules for how includes are to be ordered, but no automated enforcement.